tasks/workflow/worker.go

```diff
@@ -228,9 +228,6 @@ func (w *worker) handleExecution(t *taskExecution) {
 	}
 	if err != nil {
 		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelERROR, t.targetID).Registerf("%v", err)
```

Comment:
I think we could still have details in debug level WDYT?

```suggestion
		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelERROR, t.targetID).Registerf("%v", err)
		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelDEBUG, t.targetID).Registerf("%+v", err)
```

Reply:
Ok

Reply:
Done !